tests/unit_tests/layout/test_layout_array.py

```diff
@@ -39,7 +39,7 @@ def south_layout_center_data_dict():
 
 @pytest.fixture
 def layout_array_north_four_LST_instance(
-    north_layout_center_data_dict, manual_corsika_dict_north, io_handler, db_config
+    north_layout_center_data_dict, manual_corsika_dict_north, db_config
```

Comment:
I think we sometimes pass the `io_handler` fixture because it sets the temporary output directories. Did you check that after removing this argument, no files/output is written to the local directory?

Reply:
Just checked this again and no files are written to the local directories for these tests.